lib/gracenote/response/base.rb

```diff
@@ -14,7 +16,14 @@ def initialize(response)
       end
 
       def [](attr)
-        @response[attr]
+        values = wrap_array(@response[attr]).map do |value|
+          if value.kind_of?(Hash) && value.has_key?("__content__")
```

Comment:
__[rubocop]__
* Prefer `Object#is_a?` over `Object#kind_of?`.
* `Hash#has_key?` is deprecated in favor of `Hash#key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)